### PR TITLE
#216 make support requests index sortable

### PR DESCRIPTION
**Pull request by misternu**
## Changelog
Support request index is now sortable using datatables
Each row is now a clickable and takes you to the support request

## Link to issue:  
https://github.com/MidwestAccessCoalition/lockbox_rails/issues/216

## Relevant Screenshots: 
Screen Shot 2019-10-27 at 2 16 22 PM

## Are you ready for review?:

- [ ] Added relevant tests
- [x] Linked PR to the issue
- [ ] Added notes for QA/special notes
- [x] Added revelant screenshots

### CODE REVIEW

File: app/javascript/src/pending_support_requests_table.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+};
+
+document.addEventListener('turbolinks:load', () => {
+  let table = $('#supportRequestsTable');
+  if (table === null) {
+    return false;
+  }
+  $('#supportRequestsTable').dataTable({
+    paging: false,
+    searching: false,
```
Comment (BintLopez) [inline app/javascript/src/pending_support_requests_table.js]: not for this issue -- but does adding searching: true make it super easy to add a search bar for the table? or does it add a bunch of extra work?

File: app/views/support_requests/index.html.erb [hunk 2]
```diff
@@ -11,29 +11,24 @@
...
-            <td>$<%= support_request.amount.to_s %></td>
             <td><%= support_request.lockbox_partner.name %></td>
-            <td>
-              <%= link_to [support_request.lockbox_partner, support_request] do %>
-                View Details
-                <i class="fa fa-long-arrow-right" aria-hidden="true"></i>
-              <% end %>
-            </td>
+            <td class="blue-amount">$<%= support_request.amount.to_s %></td>
+            <td>Coordinator Name</td>
```
Comment (BintLopez) [inline app/views/support_requests/index.html.erb]: This is a placeholder still... was this intentional?

File: app/views/support_requests/index.html.erb [hunk 3]
```diff
@@ -11,29 +11,24 @@
...
-            <td>$<%= support_request.amount.to_s %></td>
             <td><%= support_request.lockbox_partner.name %></td>
-            <td>
-              <%= link_to [support_request.lockbox_partner, support_request] do %>
-                View Details
-                <i class="fa fa-long-arrow-right" aria-hidden="true"></i>
-              <% end %>
-            </td>
+            <td class="blue-amount">$<%= support_request.amount.to_s %></td>
+            <td><%= support_request.user.name %></td>
```
Comment (BintLopez) [inline app/views/support_requests/index.html.erb]: if the user name is blank, let's display the user ID

**Comment by misternu** [comment]
I had this commit last week but it wasn't up sorry

**Pull request merged**